code/modules/uplink/uplink_items.dm

```diff
@@ -615,14 +617,14 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	desc = "An additional 28-round 10mm magazine suitable for use with the C-20r submachine gun. \
 			Loaded with incendiary rounds which are weak in impact, but carry a payload that will ignite the target."
 	item = /obj/item/ammo_box/magazine/smgm10mm/fire
-	cost = 2
+	cost = 3
 
 /datum/uplink_item/ammo/smg/zzz
 	name = "10mm SMG Soporific Magazine"
 	desc = "An additional 28-round 10mm magazine suitable for use with the C-20r submachine gun. Loaded with soporific rounds that put the target to sleep. \
 			Remember that it takes about three shots to actually put the target to sleep, so plan accordinlgy."
 	item = /obj/item/ammo_box/magazine/smgm10mm/soporific
-	cost = 2
+	cost = 3
```

Comment:
Make this way higher, its a 3 hit sleep, thats pens all armor do to how sleep works